add assistant blog post (#30771)
supabase/supabase: apps/www/_blog/2024-12-02-supabase-assistant.mdx

---

> title: 'Introducing Assistant V2'

Reviewer: ```suggestion
title: 'Supabase AI Assistant v2'
```

description: An evolution of how we approach AI within the Supabase dashboard'
author: saxon_fletcher
image: launch-week-13/assistant/header.png
thumb: launch-week-13/assistant/header.png
categories:
  - developers
  - postgres
tags:
  - AI
  - postgres
date: '2024-12-02'
toc_depth: 3
---

> Today we are announcing an update to the our AI capabilities within the dashboard by introducing a global Assistant with several new abilities:

Reviewer: ```suggestion
Today we releasing Supabase Assistant v2 in the dashboard - a global assistant with several new abilities:
```

> - Postgres schema design
> - Data queries and charting
> - Error debugging
> - Postgres RLS Policies: create and edit
> - Postgres Functions: create and edit
> - Postgres Triggers: create and edit
> - SQL to supabase-js conversion

Reviewer: ```suggestion
1. Postgres schema design
1. Data queries and charting
1. Error debugging
1. Postgres RLS Policies: create and edit
1. Postgres Functions: create and edit
1. Postgres Triggers: create and edit
1. SQL to `supabase-js` conversion
```
